server/applicationPattern/applicationPattern/onfModel/services/LogicalTerminationPointServices.js

```diff
@@ -620,42 +620,37 @@ function updateOperationClientInterface(operationClientUuid, operationName) {
  * @description This function creates or updates a operation client interfaces for the provided input operation name list.
  * @param {String} httpClientUuid : uuid of the http-client, the value should be a valid string 
  * in the pattern '-\d+-\d+-\d+-http-c-\d+$'
- * @param {list} operationClientNameList : list of operation names
+ * @param {Map} operationClientNameList : list of (new) operation names (keys) with their UUIDs (values)
  * @return {Promise} object {configurationStatusList}
  **/
 function createOrUpdateOperationClientInterface(httpClientUuid, operationClientNameList) {
```

Comment:
Modifying the object type to map looks good in this area of code. But , the calling services are still not updated to formulate the operationClientNameList to a map.
For example , 
the "registerApplication" individual service uses this function  "createOrUpdateOperationClientInterface".
But , the object in the calling function is still a list and not a map.

registerApplication --> createOrUpdateApplicationInformationAsync --> updateLogicalTerminationPointInstanceGroupAsync --> createOrUpdateOperationClientInterface is failing because of this reason.

Also , the following services in registry office is failing , 
TypeError: Failed to fetch
**Individual Services**
a.	/v1/register-application
**Basic Services**
a.	/v1/redirect-service-request-information
b.	/v1/redirect-oam-request-information
c.	/v1/inquire-oam-request-approvals

Internal server Error  
a.	/v1/notify-withdrawn-approvals
b.	/v1/notify-approvals
c.	/v1/notify-deregistrations
d.	/v1/inquire-application-type-approvals

Can we have something like this , 
1. Make the existing service as a deprecated one
2. Introduce this new service.
3. create separate issues for the services that are impacted with this changes and resolve them in concrete application.

Also kindly let me know your views. Many thanks.

Reply:
Hi Prathiba,

I am aware that other applications are failing, because they are not updated to this API change. However I would like you first to OK this change (no need to merge yet) before I make changes in other applications. When you OK the API change here, I will make pull requests for other applications. I have changed slightly the operationsList, so please take a look.

Thanks,
Martin